syntax = "proto3";

service IPService 
{
    rpc AddIP (IPRequest) returns (Response);
    rpc RemoveIP (IPRequest) returns (Response);
    rpc GetIPs (ServiceRequest) returns (IPList);
}


message IPRequest 
{
    string service = 1;
    string ip = 2;
}


message ServiceRequest 
{
    string service = 1;
}


message IPList 
{
    string ips = 1;
}


message Response 
{
    string message = 1;
}
